test/inRange.js

```diff
@@ -0,0 +1,76 @@
+import chai from 'chai';
+import * as R from 'ramda';
+
+import * as RA from '../src/index';
+import eq from './shared/eq';
+
+const supportsMinMaxSafeInt = R.all(RA.isNotUndefined, [
+  Number.MAX_SAFE_INTEGER,
+  Number.MIN_SAFE_INTEGER,
+]);
+
+describe('inRange', function() {
+  it('tests whether a value falls within supplied range', function() {
+    const f = RA.inRange(5, 10);
+    eq(f(5), true);
```

Comment:
Can we insert empty line here or prettier will remove it ?

Reply:
Prettier will remove.